DiscordBot/Program.cs

```diff
@@ -40,9 +47,57 @@ private static async Task MainAsync()
             commands.RegisterCommands<EventCommands>();
 
             discord.UseInteractivity(new InteractivityConfiguration());
+            discord.MessageReactionAdded += OnMessageReactionAdded;
```

Comment:
Why `+=`? 🙂

Reply:
OnMessageReactionAdded is the handler for the MessageReactionAdded event. Is there a different way I should do this?

Reply:
I would expect just an assignment rather than `+=`? 🙂

Reply:
I get a compiler error with that:
"The event 'MessageReactionAdded' can only appear on the left hand side of += or -="

Reply:
I see! I guess it must be specific to the discord library api then. Is there anything in the docs about it? Either way, if the discord library enforces it then fair enough! 🙂

Reply:
Yup, there's an article on Events on the library website [here](https://dsharpplus.github.io/articles/beyond_basics/events.html).